cmd/root.go

```diff
@@ -11,11 +11,52 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
 
-func newRoot(ctx context.Context) *cobra.Command {
+func NewRoot(ctx context.Context) *cobra.Command {
```

Comment:
Any reason why you're exporting this function? 🤔

Reply:
The man page exporter needs it. Which is in a different package.

Reply:
yup